en/stacks/tpm/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: TPM
+---
+
+# Trusted Platform Module (TPM)
+
+There are currently two different snaps that help manage TPM under Ubuntu Core Stacks
```

Comment:
For now, I added a link to the Wikipedia page for TPM. We should improve on this for the future when appropriate.